src/components/common/notification-center.svelte

```diff
@@ -46,6 +58,39 @@
       <div class="title">No notifications</div>
     </div>
   {:else}
-    <p>List notifications here</p>
+    <SimplebarList classname="notification-list">
+      {#each notifications as notification (notification.id)}
+        <li class="notification">
+          <span class="content">
+            {#each getNotificationContent(notification) as fragment}
+              {#if typeof fragment === 'string'}
+                {fragment}
+              {:else if 'bold' in fragment}
+                <strong>{fragment.text}</strong>
+              {:else}
+                <a href="{fragment.url}" rel="prefetch">
+                  {fragment.text}
+                </a>
+              {/if}
+            {/each}
+          </span>
+          <Button
+            isRound
+            tooltip="Mark as read"
+            classname="mt"
+            on:click={() => readNotification(notification.id)}
+          >
+            <svg src="images/icons/check.svg" class="icon" />
+          </Button>
+          <time datetime={notification.timestamp}>
+            {fecha.format(new Date(notification.timestamp), 'HH:mm, DD.MM')}
```

Comment:
No need for fecha at all. Same thing can be achieved using `new Date().toLocaleString('ru', {
    hour: '2-digit',
    minute: '2-digit',
    day: '2-digit',
    month: '2-digit',
})`

Reply:
Yeah. Previously `fecha` was used in another module which I just refactored, so we might remove this dependency altogether